### PR TITLE
Add firefox/nightly/whatsnew.ftl

### PR DESCRIPTION
**Pull request by alexgibson**
Ref: https://github.com/mozilla/bedrock/pull/9186

### CODE REVIEW

File: configs/pontoon.toml [hunk 1]
```diff
@@ -115,6 +115,16 @@ locales = [
     "zu",
 ]
 
+[[paths]]
```
Comment (alexgibson) [inline configs/pontoon.toml:118]: @peiying2 this is the list of supported locales for this page as per langchecker: https://github.com/mozilla-l10n/langchecker/blob/master/app/config/sources.inc.php#L258-L268

Is this restriction still wanted / needed, or would it be better to open this up for all locales?

I notice the nightly /firstrun page is supported more broadly in comparison: https://github.com/mozilla-l10n/langchecker/blob/master/app/config/sources.inc.php#L255-L257

Comment (peiying2) [reply configs/pontoon.toml:118]: Both files were created 4-5 years ago, with nightly/whatsnew.lang updated earlier this year. Unless it is tied to the actual released locales to nightly, I am ok to open it to more locales.

@flodolo need your help with the background info.

Comment (peiying2) [reply configs/pontoon.toml:118]: @alexgibson we can open it to all locales. I re-read the setting in langchecker, it is open to all Firefox locales, but communities needs to opt-in to make it available for localization. nightly_firstrun.lang is an older file and it was open to all locales. Not sure what drove the change.

**Comment by alexgibson** [comment]
> we can open it to all locales.

Sure, I'll update and remove the path config here.

I'm trying to piece together a full list of locales to apply for this migration. Looking here, I think I have everybody? https://l10n.mozilla-community.org/langchecker/?locale=all&website=0&file=firefox/nightly_whatsnew.lang

File: configs/pontoon.toml (hunk 1 above)
Comment (flodolo) [reply configs/pontoon.toml]: As far as I can tell, the locales available in Nightly are a superset of the locales available in mozilla.org

The limitation comes from a time where we had limited locales on Nightly (we had Aurora, which was the focus for localization), but it doesn't make sense anymore these days, for both whatsnew and firstrun.

**Comment by peiying2** [comment]
https://github.com/mozilla-l10n/langchecker/blob/master/app/config/sources.inc.php#L258-L268 and https://l10n.mozilla-community.org/langchecker/?locale=all&website=0&file=firefox/nightly_whatsnew.lang are the same, the latter is a status dashboard, and the langchecker file is the setting. The two lists should match. 

If I approve this PR, should I merge or should I wait for you to finish up work else where

**Comment by alexgibson** [comment]
> If I approve this PR, should I merge or should I wait for you to finish up work else where

If this looks look then please feel free to merge. I can then see to the bedrock PR and get that reviewed. Thanks

**Pull request merged**